cmd/content-sources/main.go

```diff
@@ -3,18 +3,31 @@ package main
 import (
 	"log"
 
+	"github.com/content-services/content-sources-backend/pkg/db"
 	"github.com/gin-gonic/gin"
 )
 
 func main() {
+	var err error
+
+	err = db.Connect()
+	if err != nil {
+		log.Fatalf("%v", err)
+	}
+
+	err = db.MigrateDB(db.GetUrl(), "up")
+	if err != nil {
+		log.Fatalf("%v", err)
+	}
```

Comment:
I'd get rid of the migration on startup.  I think we'll use an 'init container' to accomplish this.   This also causes the server to die if its fully migrated